src/Libraries/Microsoft.Extensions.Http.Diagnostics/Logging/HttpClientLoggingTagNames.cs

```diff
@@ -14,47 +14,47 @@ public static class HttpClientLoggingTagNames
     /// <summary>
     /// HTTP Request duration.
     /// </summary>
-    public const string Duration = "duration";
+    public const string Duration = "Duration";
 
     /// <summary>
     /// HTTP Host.
     /// </summary>
-    public const string Host = "httpHost";
+    public const string Host = "server.address";
```

Comment:
note that HTTP Host header is a combination of host and port. If you want to capture the host header as it, it's best to use `http.request.header.host` attribute instead.

Also, `server.address` is not always derived from the host header. It may be derived from `Forwareded` or `X-Forwarded-Host`.

Reply:
We are reading the host from a request's Host property which is the name of the host without port information:

https://github.com/dotnet/extensions/blob/d74d9f4915cf6a1bdd38f784fcbda2861312bd8e/src/Libraries/Microsoft.Extensions.Http.Diagnostics/Logging/Internal/HttpRequestReader.cs#L93

So, as I understand, in our case `Host` is exactly `server.address` dimensions from the semantic-convetions.